### PR TITLE
Move maxRetries config to aux db

### PR DESCRIPTION
**Pull request by goughes**

### CODE REVIEW

File: src/python/WMComponent/ErrorHandler/ErrorHandlerPoller.py [hunk 1]
```diff
@@ -65,7 +65,15 @@ def __init__(self, config):
                                      dbinterface=myThread.dbi)
         self.changeState = ChangeState(self.config)
 
-        self.maxRetries = self.config.ErrorHandler.maxRetries
+        if hasattr(self.config, "Tier0Feeder"):
```
Comment (goughes) [inline src/python/WMComponent/ErrorHandler/ErrorHandlerPoller.py]: @ticoann Let me know if this is necessary or if I can remove it

Comment (goughes) [reply src/python/WMComponent/ErrorHandler/ErrorHandlerPoller.py]: Meant to click line 70

**Comment by ticoann** [comment]
Sorry Erik, Now there is conflict on the patch, due to other patch update. Could you update the patch. Patch looks good to me. I can test it when patch is updated.

**Comment by amaltaro** [comment]
Are we using ReqMgr aux db for T0 agents?

**Comment by ticoann** [comment]
test this please.

**Comment by ticoann** [comment]
test this please

**Comment by ticoann** [comment]
test this please

**Comment by ticoann** [comment]
test this please

**Comment by amaltaro** [comment]
test this please

**Comment by ticoann** [comment]
@goughes, Erik, could you squash the commit?
Error seems to be temporary error. So I am running the test again.

**Comment by ticoann** [comment]
test this please

**Comment by ticoann** [comment]
test this please

**Comment by ticoann** [comment]
test this please

File: etc/WMAgentConfig.py [hunk 2]
```diff
@@ -228,7 +228,6 @@
 config.ErrorHandler.namespace = "WMComponent.ErrorHandler.ErrorHandler"
 config.ErrorHandler.componentDir = config.General.workDir + "/ErrorHandler"
 config.ErrorHandler.logLevel = globalLogLevel
-config.ErrorHandler.maxRetries = maxJobRetries
```
Comment (amaltaro) [inline etc/WMAgentConfig.py]: @goughes can you please add this line back? I _think_ we need it for the T0 case.

**Comment by hufnagel** [comment]
FYI, T0 doesn't really need maxRetries to be in the WMAgent config template. We override it anyways.

https://github.com/dmwm/T0/blob/master/etc/Tier0Config.py#L27

File: etc/WMAgentConfig.py [hunk 3]
```diff
@@ -55,7 +55,7 @@
              'cmssrm-fzk.gridka.de', 'ccsrm.in2p3.fr', 'srmcms.pic.es', 'cmssrmdisk.fnal.gov']
 
 # Job retry information.  This includes the number of times a job will be retried and
-# for how long it will sit in cool off.
+# for how long it will sit in cool off. maxJobRetries only used in T0 agent
 maxJobRetries = {'default': 3, 'Merge': 4, 'LogCollect': 2, 'Cleanup': 2, 'Harvesting': 2}
```
Comment (amaltaro) [inline etc/WMAgentConfig.py]: Given that @hufnagel said T0 overwrites it, how about we completely delete it from this config then?

**Pull request merged**